### PR TITLE
Update the windows guide section

### PR DESCRIPTION
**Pull request by melund**

### CODE REVIEW

File: docs/windows.rst [hunk 1]
```diff
@@ -4,26 +4,43 @@ Windows Guide
...
-To install xonsh on Windows, first install `Python v3.4+`_ from
-http://python.org .
 
-Next, install the other dependencies via ``pip``:
+The easy way
+----------------
+
+The easiest way to install xonsh on windows is through the `Anaconda Python Distribution`_ and the conda package manager. 
+
+.. note:: Be sure to install the version with Python3.4 or 3.5. Xonsh does is not supported on legacy Python (2.7).
```
Comment (scopatz) [inline docs/windows.rst]: Maybe you can say `Python v3.4 and later`, so that this doesn't go out of date with v3.6 :)

**Comment by scopatz** [comment]
Thanks for the improvements, @melund! One minor comment.

**Comment by melund** [comment]
I thought I better put this little thing in before #683 ... I was not really worth a prime issue number.

**Comment by scopatz** [comment]
> I thought I better put this little thing in before #683 ... I was not really worth a prime issue number.

hahaha

**Comment by scopatz** [comment]
Thanks @melund!

File: docs/windows.rst [hunk 2]
```diff
@@ -4,26 +4,43 @@ Windows Guide
...
-To install xonsh on Windows, first install `Python v3.4+`_ from
-http://python.org .
 
-Next, install the other dependencies via ``pip``:
+The easy way
+----------------
+
+The easiest way to install xonsh on windows is through the `Anaconda Python Distribution`_ and the conda package manager. 
+
+.. note:: Be sure to install the version with Python3.4 or later. Xonsh does is not supported on legacy Python (2.7).
```
Comment (BYK) [inline docs/windows.rst]: `does is not`?

Comment (melund) [inline docs/windows.rst]: Ups.. I will fix it in a new PR as soon as someone grabs the #683 issue...

**Pull request merged**